expressjs-backend/models/task-services.js

```diff
@@ -172,4 +185,5 @@ exports.editTask = editTask;
 exports.getCategories = getCategories;
 exports.getCategoriesOfUser = getCategoriesOfUser;
 exports.completeTask = completeTask;
+exports.deleteCompeteByUser = deleteCompeteByUser;
```

Comment:
did you catch this typo too?

Reply:
Fixed that one when Cameron mentioned the other typo. Also fixed the one in backend.js, I'll push those changes soon